reana_resources_k8s/templates/deployments/job-controller-template.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         volumeMounts:
         - name: svaccount
           mountPath: "/var/run/secrets/kubernetes.io/serviceaccount"
+        env:
+        - name: STORAGE_BACKEND
```

Comment:
Shall we go for `REANA_SHARED_STORAGE_BACKEND` to be more explicit? Then the in-module vars would be something like `SHARED_STORAGE_PATHS`, `SHARED_STORAGE_MOUNT_POINT` etc.. We may have in the future some storage for storing results such as `REANA_RESULT_STORAGE_*` @tiborsimko @hjhsalo